src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -49,13 +54,13 @@ const ChatItemMyTextV2 = ({
   const renderIcon = React.useCallback(() => {
     if (status === ChatStatus.PENDING)
       return (
-        <View style={styles.icon}>
+        <View style={styles.iconNewLine}>
           <IconChatClockGrey color={colors.silver} width={12} height={12} />
         </View>
       );
 
     return (
-      <View style={styles.icon}>
+      <View style={styles.iconNewLine}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in style from `styles.icon` to `styles.iconNewLine` might be due to the need to accommodate the display of attachments along with the message. Without seeing the actual styles, it's hard to say if this change is appropriate or not.
```diff
- <View style={styles.icon}>
+ <View style={styles.iconNewLine}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->